spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/actions/RewriteManifestsSparkAction.java

```diff
@@ -90,13 +94,13 @@ public class RewriteManifestsSparkAction
   public static final String USE_CACHING = "use-caching";
   public static final boolean USE_CACHING_DEFAULT = false;
 
+  private List<String> partitionSortColumns = null;
```

Comment:
nit: we follow `public static final` -> `private static final` -> `private final` -> `private` to order variables

Reply:
Is this a problem? I have these attributes as being set to `null` but able to be redefined. So I think `private` is correct here, its inline with the other attributes in this class.